MapboxCoreNavigation/RouteController.swift

```diff
@@ -360,6 +359,10 @@ extension RouteController: CLLocationManagerDelegate {
         
         self.lastRerouteLocation = location
         
+        if let accessToken = routeProgress.route.accessToken, let apiEndpoint = routeProgress.route.apiEndpoint, let host = apiEndpoint.host {
+            directions = Directions(accessToken: accessToken, host: host)
+        }
+        
         routeTask = directions.calculate(options, completionHandler: { [weak self] (waypoints, routes, error) in
```

Comment:
Doh, didn't see your comment before I merged. Yeah, this is the only spot we're currently using it. 

I still see an advantage to a developer being able to access the `directions` property on RouteController if they want to use it for their own routing/rerouting logic.